bcda/models/alr.go

```diff
@@ -25,12 +25,24 @@ type AlrMetaData struct {
 	Timestamp time.Time
 }
 
+// Wrap AlrMBIs as []string to ensure not any []string is accepted
+// See repository.go for more info... particularly GetAlrMBIs func
+type AlrMBIs struct {
+	MBIS            []string
+	Metakey         int64
+	CMSID           string
+	TransactionTime time.Time
+}
+
 // There is no AlrJobs struct because ALR uses Job struct from BFD
 type JobAlrEnqueueArgs struct {
-	ID         uint
-	CMSID      string
-	MBIs       []string
-	BBBasePath string
-	LowerBound time.Time
-	UpperBound time.Time
+	ID              uint
+	CMSID           string
+	MBIs            []string
+	ResourceType    []string // Currently Not Used
+	MetaKey         int64
+	BBBasePath      string
+	LowerBound      time.Time // Currently Not Used
+	UpperBound      time.Time // Currently Not Used
+	TransactionTime time.Time
```

Comment:
New fields needed in the struct for new way of querying. There are fields we do not use, as marked above. We'll look into removing in a later ticket.